mms/model_service/model_service.py

```diff
@@ -21,9 +20,11 @@
 from mms.log import get_logger
 from mms.mxnet_model_service_error import MMSError
 from mms.utils.model_server_error_codes import ModelServerErrorCodes as err
-
+from mms.metrics.metrics_store import MetricsStore
 logger = get_logger()
-
+PREPROCESS_METRIC = 'MMS_Worker_Preprocess_time_batch'
```

Comment:
Why not choose CamelCase? instead of underscore seperated values? Wouldn't that be better to read into a Java JSON object?

Reply:
Agreed will change it